client/src/lib/dependency-injection/inject.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import injectorContext from './injectorContext';
 
-const inject = (Component, dependencies, mapDependenciesToProps) => {
+const inject = (dependencies, mapDependenciesToProps, contextArg) => (Component) => {
```

Comment:
I don't think the asset-admin has been updated with this new signature?

please add JSDoc block for the HOC at least